resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerBackendUtil.scala

```diff
@@ -17,10 +17,14 @@
 
 package org.apache.spark.scheduler.cluster.mesos
```

Comment:
Out of curiosity, why do we have this file _and_ `MesosSchedulerUtils`?

Reply:
Its for the MesosScheduler's helper functions but there are methods (eg. createResource) used all over the place.